ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -98,35 +98,35 @@
 
 (defn- related-url-validator
   "Return a validator that checks a ContentType, Type, and Subtype keyword combo
-   plus Format and MimeType in GetData for Related URL field which can be inside
-   a ContactInformation or be a standalone field. ContactInformation can themselves
-   be found in DataCenters, ContactGroups, and ContactPersons."
-  [kms-index]
-  {:RelatedUrls 
-   [(match-related-url-kms-keywords-validations kms-index)
-    (v/every {:GetData {:MimeType (match-kms-keywords-validation-single
-                                   kms-index
-                                   :mime-type
-                                   msg/mime-type-not-matches-kms-keywords)}})]})
+   if warn? is set to false, otherwise will check MimeType in GetData for Related URL
+   field. All of which can be inside a ContactInformation or be a standalone field.
+   ContactInformation can themselves be found in DataCenters, ContactGroups, and ContactPersons."
+  [kms-index warn?]
+  (if (not warn?)
```

Comment:
double negative? so I send false to get a true? Why not rename to something like report-as-error and when it is true you do the thing? There is also an if-not function, but I think you can make it read better.

Reply:
`(not warn?)` was used in other parts of the file, I can see how in my case its not as readable though I can change that

Reply:
you can get a second opinion, I will defer if I'm the only one

Reply:
The logic is right. Although I don't like using the flag because it's a little confusing. I messaged Erik about it.